README.md

```diff
@@ -12,6 +6,41 @@ A graphical user interface for controlling and monitoring an interferometer devi
 [![Test and build](https://github.com/ImperialCollegeLondon/FINESSE/actions/workflows/ci.yml/badge.svg)](https://github.com/ImperialCollegeLondon/FINESSE/actions/workflows/ci.yml)
 [![codecov](https://codecov.io/gh/ImperialCollegeLondon/FINESSE/graph/badge.svg?token=4UILYHPMJT)](https://codecov.io/gh/ImperialCollegeLondon/FINESSE)
 
+# FINESSE
+
+FINESSE is open-source graphical software to control a spectrometer system developed at

```

Comment:
This reads a bit strangely to me. Would something like "FINESSE is _an_ open-source graphical software" work better?

Reply:
I know what you mean... How about something like:

```
FINESSE is software for controlling a spectrometer system developed by Imperial College London's [Space and Atmospheric Physics group], written in Python and with a graphical interface using the [PySide6 Qt bindings].
```

I think it might be good to mention some of the technical stuff up front and I'm also not sure we need to mention that it's open-source here as it should be obvious.